src/ant/src/AntennaChecker.cc

```diff
@@ -218,10 +220,22 @@ void AntennaChecker::initAntennaRules()
                                   plus_diff_factor,
                                   diff_metal_reduce_factor};
     layer_info_[tech_layer] = layer_antenna;
-  }
 
-  block_ = db_->getChip()->getBlock();
-  dbu_per_micron_ = block_->getDbUnitsPerMicron();
+    double wire_width = dbuToMicrons(tech_layer->getWidth());
```

Comment:
Why both with dbuToMicrons if you are just going to == 0?  Same with thickness

make const  (I'm quite pedantic about const).